cloudformation/content-authorisation-proxy.cf.yaml

```diff
@@ -0,0 +1,250 @@
+         mkdir /etc/gu
+         CONF_DIR=/etc/content-authorisation-proxy
+         wget -N --directory-prefix=/home/ubuntu/.ssh https://s3-eu-west-1.amazonaws.com/subscriptions-dist/${Stack}/authorized_keys &
+         apt-get -y update
+         apt-get -y upgrade
+         aws --region ${AWS::Region} s3 cp s3://subscriptions-dist/${Stack}/${Stage}/content-authorisation-proxy/content-authorisation-proxy_0.1_all.deb /tmp
+         aws --region ${AWS::Region} s3 cp s3://subscriptions-private/${Stage}/cas-proxy.private.conf /etc/gu
+         dpkg -i /tmp/content-authorisation-proxy_0.1_all.deb
+         chown content-authorisation-proxy /etc/gu/cas-proxy.private.conf
+         chmod 0600 /etc/gu/cas-proxy.private.conf
+         wget https://s3.amazonaws.com/aws-cloudwatch/downloads/latest/awslogs-agent-setup.py
+         sed -i -e "s/__DATE/$(date +%F)/" -e 's/__STAGE/${Stage}/' $CONF_DIR/logger.conf
+         python awslogs-agent-setup.py -nr ${AWS::Region} -c $CONF_DIR/logger.conf
+         systemctl enable awslogs
+         systemctl start awslogs
+        - {}
+
+  SubscriptionsAppRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Path: /
+      AssumeRolePolicyDocument:
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service:
+            - ec2.amazonaws.com
+          Action:
+          - sts:AssumeRole
+      Policies:
+      - PolicyName: PrivateBucket
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action: s3:GetObject
+            Resource: arn:aws:s3:::subscriptions-private/*
+      - PolicyName: PushLogs
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - logs:*
+            Resource:
+            - arn:aws:logs:*:*:*
+      - PolicyName: dynamoTable
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - dynamodb:PutItem
+            - dynamodb:GetItem
+            - dynamodb:UpdateItem
+            - dynamodb:DeleteItem
+            - dynamodb:BatchGetItem
+            Resource: !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/cas-auth-${Stage}'
+      - PolicyName: PushMetrics
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - cloudwatch:PutMetricData
+            Resource: '*'
+  SubscriptionsAppInstanceProfile:
+    Type: AWS::IAM::InstanceProfile
+    Properties:
+      Path: /
+      Roles:
+      - !Ref 'SubscriptionsAppRole'
+  CASProxyElasticLoadBalancer:
+    Type: AWS::ElasticLoadBalancing::LoadBalancer
+    Properties:
+      Listeners:
+      - LoadBalancerPort: '443'
+        InstancePort: 9300
+        Protocol: HTTPS
+        SSLCertificateId: arn:aws:acm:eu-west-1:865473395570:certificate/4055df40-9561-423c-962b-f78f3ce70434
```

Comment:
If the certificate is in the membership AWS account, can we use ${AWS::Region}:${AWS::AccountId} here too?